workorders.lic

```diff
@@ -606,13 +611,15 @@ class WorkOrders
         echo('out of material/deeds')
         exit
       else
+        volume = deed_ingot_volume
         fput('tap deed')
         pause
         fput("get #{@use_own_ingot_type} ingot") unless checkleft || checkright
       end
     end
-
-    volume = ingot_volume
+    if(!volume)
```

Comment:
What do you think about `volume = ingot_volume unless volume`?

Reply:
that works too :)